dev/oracle/21-slim/wait-until-healthy.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved
```

Comment:
Given that this file is basically a copy of the existing wait-script, isn't a bit funny that the rights are credited to Oracle?

Reply:
removed the copyright information.

Reply:
thx